### PR TITLE
APIGOV-10320: Add doc and images for backend http basic authn

### PR DESCRIPTION
**Pull request by alexearnshaw**
- Created a new topic for backend authn based on https://github.com/Axway/Amplify-Setup-Backend-BasicAuth
- Used a single topic instead of multiple connected topics so that it can be easily extended with other backend authn mechanisms
- Edited the topic to be more concise and to cover only the relevant information for this fairly simple procedure
- Removed the screens/animations that didn't add a lot of value to the text as these are hard to maintain. Kept 1 screen for orientation, and created a single animation to replace multiple animations  demonstrating the key steps

### CODE REVIEW

**Comment by alexearnshaw** [comment]
@sjanagam12 Please review this doc relating to https://techweb.axway.com/jira/browse/APIGOV-10320 (https://techweb.axway.com/jira/browse/APIGOV-10042).

File: content/en/docs/central/backend_authn.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: Configure back-end authentication for an API
+linkTitle: Configure back-end authentication
+weight: 5
+date: 2019-07-30
+description: Learn how to configure back-end authentication for an API which supports back-end authentication.
```
Comment (andreamussap) [inline content/en/docs/central/backend_authn.md]: This is repetitive IMHO. I'd keep just 'Learn how to configure back-end authentication for an API'

File: content/en/docs/central/backend_authn.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Objectives
+
+Learn how to secure your back-end API using HTTP Basic authentication.
+
+## Add back-end HTTP basic authentication to an API proxy
+
+The following steps demonstrate how to configure HTTP basic back-end authentication for a sample API, and how to deploy and test the API.
+
+### Step 1 - Import an API that supports back-end authentication as an API proxy
```
Comment (andreamussap) [inline content/en/docs/central/backend_authn.md]: Why are we using the prefix "Step X" in this topic if we don't use it (at least no in general)? It's unnecessary IMHO. Could change the description to ~ "The following procedure demonstrate how to configure...", and remove the prefixes.

File: content/en/docs/central/backend_authn.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Select **API Proxies** in the left navigation bar, click **Register API**,  and import an API with back-end authentication. 
+
+This example uses a sample musical instruments API with HTTP basic authentication. You can import this from: <https://39cd48c2323117d7f0ec53a347aa94d8974a2174.cloudapp-enterprise.appcelerator.com/apidoc/swagger.json>
+
+![Import an API with HTTP basic authentication](/Images/central/import-proxy.png)
+
+### Step 2 - Configure HTTP basic back-end authentication for the API proxy
+
+On the **Policies** tab change the back-end authentication policy to `HTTP Basic Auth` and enter the authentication credentials. The credentials can be a user name and password combination or only a user name with no password.
```
Comment (andreamussap) [inline content/en/docs/central/backend_authn.md]: add comma ==>  On the **Policies** tab,

File: content/en/docs/central/backend_authn.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+![Configure HTTP basic authentication details](/Images/central/APICBackendHTTPBasic1Cropped.gif)
+
+Changing the back-end authentication policy (for example, from no authentication to HTTP basic authentication) always creates a new revision of the API proxy.
+
+### Step 3 - Deploy and test the API proxy revision
+
+On the **Deployments** tab, deploy or update a runtime group with this revision of your API proxy.
+
+On the **Test Methods** tab, execute a method and it should return a `200 OK` result. This indicates a successful call of the API using HTTP basic as the back-end authentication.
```
Comment (andreamussap) [inline content/en/docs/central/backend_authn.md]: RE execute a method **and** it should return -> suggestions:
- execute a method. It should return...
- execute a method, which should return...

**Pull request merged**